docs/advanced_usage.rst

```diff
@@ -277,4 +277,37 @@ also make a switchable widget. You can use it like this:
 
         switched_widget.register('Action type 1', default=True, widget=Widget())
 
-Then instead of switching views, it switches widgets.
\ No newline at end of file
+Then instead of switching views, it switches widgets.
+
+IFrame support is views
+-----------------------------
+
+If some html page has embedded iframes, those can be covered using regular view.
+You just need to set FRAME property for it. FRAME should point out to appropriate iframe and can be xpath and whatever supported by widgetastic.
+
+Since iframe is another page, all its bits consider iframe as root. This has to be taken into account during creating object structure.
+
+If regular views and iframe views are mixed, widgetastic takes care of switching between frames on widget access. 
+User doesn't need to undertake any actions.
+
+Below is example of usage. More examples can be found in unit tests.
+
+.. code-block:: python
+
+    class FirstIFrameView(View):
```

Comment:
Maybe start your example with RegularView, that has FRAME in one of its embedded views.

Consider an example usage call as well, assuming I have a view object for RegularView, can I just call `view.FirstIFrameView.select1.read()`?

Reply:
I see now in the tests how you're creating two view objects, one for the iFrame view and one for the non-framed view. Both of these are part of the same browser page, and it follows pretty clearly that one could be nested in the other.